.eslintrc

```diff
@@ -14,7 +14,8 @@
     "consistent-return": 0,
     "comma-dangle": 0,
     "spaced-comment": 0,
-    "global-require": 0
+    "global-require": 0,
+    "no-param-reassign": ["error", {"props": false}]
```

Comment:
I think what feels weird to me is having this rule combined with using things like `.forEach` because they should be allowed to mutate objects by iteration. With this rule it would be allowed to do a `for (let i = 0; i < length; i++)` modifying loop, but because `.forEach` uses a callback it doesn't allow it, yeah?

I definitely agree not to reassign params either, but when we're trying to do a loop to mutate an array of objects, we shouldn't have to do a deep copy on everyone just to change one value. Right?